src/formatDate.js

```diff
@@ -48,9 +48,56 @@
  *
  * @returns {string}
  */
+const TYPE_OF_MONTH = 'MM';
+const TYPE_OF_DAY = 'DD';
+const TYPE_OF_YEAR_LONG = 'YYYY';
+const TYPE_OF_YEAR_SHORT = 'YY';
+const CENTURE_21 = 20;
+const CENTURE_20 = 19;
+const LIMIT_YEAR = 30;
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+  const dateArr = date.split(oldSeparator);
```

Comment:
Try to choose a more meaningful naming.
```suggestion
  const oldSplittedDate = date.split(oldSeparator);
```